### PR TITLE
Add Landscap.io into project

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This Pull Request adds Landscape - hosted continuous Python code metrics

### CODE REVIEW

File: .landscape.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Include warnings about missing documentation?dd
```
Comment (TomasTomecek) [inline .landscape.yaml]: dd

Comment (phracek) [inline .landscape.yaml]: New commit fixes the typo.

**Comment by ignatenkobrain** [comment]
why not sideci?

**Comment by phracek** [comment]
@ignatenkobrain What about to create a Pull Request? We can use several CI's.

The plan is Travis CI and Landscape for now.

**Pull request merged**